code/__HELPERS/game.dm

```diff
@@ -7,6 +7,8 @@
 
 #define Z_TURFS(ZLEVEL) block(locate(1,1,ZLEVEL), locate(world.maxx, world.maxy, ZLEVEL))
 
+#define HOLIDAY_ACTIVE(holiday)  SSevent.holidays && SSevent.holidays[##holiday]

```

Comment:
What about this is or isn't C compliant? (rather, what made you comment?) everything I do and tell people to do with the preprocessor (including the ```#``` and ```##``` rules) are listed in the F1 reference of DreamMaker.